### DISCUSSION TITLE
research station doesn't have a proper brokensprite (outpost oxygen generator also)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15415] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

:barodev:

### Reproduction steps

:barodev:

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by NotWendy** [comment]
The outpost oxygen generator also doesn't have a proper broken sprite, it's just a cracked wall texture slapped on top of the regular sprite. A placeholder, essentially.

**Reply by girlyguppy** [reply]
Concerning....

**Comment by Regalis11** [comment]
This has now been addressed in our private repo, the changes will be included in the next unstable build and the upcoming release.
